MvvmCross.Android.Support/V7.RecyclerView/MvxRecyclerView.cs

```diff
@@ -33,6 +28,19 @@ public MvxRecyclerView(Context context, IAttributeSet attrs, int defStyle)
         {
         }
 
+        [Preserve(Conditional = true)]
+        protected MvxRecyclerView(IntPtr javaReference, JniHandleOwnership transfer)
+            : base(javaReference, transfer)
+        {
+        }
+
+        [Preserve(Conditional = true)]
+        protected static void LinkerPleaseInclude(MvxRecyclerView list)
```

Comment:
I don't think this should go here, but rather in a LinkerPleaseInclude file.

Reply:
I do not agree. The "Conditional" property indicates to the linker that if the MvxRecyclerView is used, then it should preserve the method. If not, it should optimize it away.

There are 3 advantages to have it here. 1) you don't have to add it elsewhere. 2) If something else should not be optimized away, or if in the future something is not required anymore, it will be in the same source file. 3) If RecyclerView is not used, it will be optimized away and not kept.

Reply:
@softlion i see your point, but if we do it this way we should probably change it all over the code base. Maybe you can remove this for now, and add a new issue to pick this up separately?

Reply:
We should start at some point. Otherwise you will never start.